### ISSUE TITLE
[BUG] SMS Server API

### ISSUE THREAD

**Issue by xmha97** [Waboodoo/HTTP-Shortcuts#317]
Hi, I can't use API of [SMS Server](https://github.com/umer0586/AndroidSMSServer) app using HTTP Shortcuts app.

But if I export cURL command, this works:

```
curl http://192.168.2.5:8080/sendSMS -X POST -H "Content-Type: application/x-www-form-urlencoded" -d phone=09360539854 -d message=Text
```

Using Termux (works)
![Screenshot_2022-11-27-20-21-43-250_com.google.android.apps.messaging.jpg](https://user-images.githubusercontent.com/23485114/204149166-4ff941ed-4e80-4d55-8848-6560047f0a9f.jpg)

Using HTTP Shortcuts (doesn't work)
![Screenshot_2022-11-27-20-22-05-196_ch.rmy.android.http_shortcuts.jpg](https://user-images.githubusercontent.com/23485114/204149172-84b43272-b973-4550-b21d-b56fa424876e.jpg)

**Comment by Waboodoo** [comment]
This seems to be a combination of a bug in the SMS Server app and a bug in the HTTP Shortcuts app:
- The SMS Server app expects the Content-Type header to not contain a charset, and therefore rejects the request from the HTTP Shortcuts app which always includes the charset. This can be worked around by explicitly adding a header in the HTTP Shortcuts app without the charset.
- That workaround however triggers a bug in the HTTP Shortcuts app, as it now no longer includes the request parameters in the request body due to the custom header which changes the content-type. I will fix that bug now by loosening the content-type check.

**Comment by xmha97** [comment]
Thanks.

**Comment by Waboodoo** [comment]
Version 2.26.0 is now available for beta testing, which includes the fix for this issue. You can join the beta program here: https://play.google.com/apps/testing/ch.rmy.android.http_shortcuts

**Comment by xmha97** [comment]
Thanks.

**Comment by Waboodoo** [comment]
This should now be fixed with the release of version 2.26.0. You just have to explicitly define the content-type header, because the SMS Server API rejects the one that HTTP Shortcuts uses by default, due to it containing the charset declaration.
